Twig/TwigExtension.php

```diff
@@ -38,6 +39,13 @@ public function __construct(
         $this->doctrine = $doctrine;
     }
 
+    public function getFilters()
+    {
+        return [
+            new TwigFilter('boolean', [$this, 'booleanFilter']),
```

Comment:
It would probable make more sense with a `yes_no` filter as we may need to actually display `Sandt` eller `Falsk` in the future.

Reply:
Fixed in: https://github.com/aakb/kontrolgruppen-core-bundle/pull/40/commits/b551f47a713f1235f36f547f5e5773d9737d1adc